Bangazon/Controllers/ProductType.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
```

Comment:
Rename this file to ProductTypeController

Reply:
Everything is ready to go!